nextflow_schema.json

```diff
@@ -510,6 +510,17 @@
                         "true"
                     ]
                 },
+                "unmatched_action": {
+                    "type": "string",
+                    "description": "Action to be taken if peptide sequences cannot be matched to any protein: 1) raise an error; 2) warn (unmatched PepHits will miss target/decoy annotation with downstream problems); 3) remove the hit. (default: 'error' valid: 'error', 'warn', 'remove')",
+                    "default": "error",
+                    "fa_icon": "far fa-check-square",
+                    "enum": [
+                        "error",
+                        "warn",
+                        "remove"
```

Comment:
You do not support "remove" in your ifcase in the main.nf
Please add it. You do not need the "def if case" actually.

Reply:
Should we support remove?

Reply:
Supported !!!!